IrisSoftware/main.py

```diff
@@ -39,28 +39,39 @@ def __init__(self) -> None:
 
         self.processingThread: threading.Thread
 
+        self.interpolator = Interpolator()
+        self.isCalibrated = False
+
         # Load calibration data
         if os.path.exists(CALIBRATION_FILE_NAME):
             self.isCalibrated = True
-            with open(CALIBRATION_FILE_NAME, "rb") as handle:
-                # TODO: train screen coords interpolator
-                pass
+            self.interpolator.calibrateInterpolator(CALIBRATION_FILE_NAME)
 
     def detectBlink(self, eyeCoords, blinkDuration) -> any:
         pass
 
     def moveMouse(self, screenX, screenY):
         '''Move the mouse to the given screen coordinates, moving smoothly over multiple frames'''
-        if self.lastCursorPos == None:
+        if self.state.lastCursorPos == None:
```

Comment:
```suggestion
        if self.state.lastCursorPos is None:
```

This is just a suggestion from pylint. In addition, we should add an expected type for `lastCursorPos` inside of the `State` class.